var/spack/repos/builtin/packages/ispc/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
```

Comment:
Shouldn't the `# ----` prelude be removed?

Reply:
gosh.. you're too fast! I didn't even have time to `git push -f`

Reply:
I am sure you meant `git push --force-with-lease` right?

Reply:
eeehm.. but hey, i just learned how to make an alias in git!
```
git config --global alias.pushf "push --force-with-lease"
```